src/Chainweb/Chainweb.hs

```diff
@@ -164,13 +169,34 @@ instance FromJSON (TransactionIndexConfig -> TransactionIndexConfig) where
 pTransactionIndexConfig :: MParser TransactionIndexConfig
 pTransactionIndexConfig = pure id
 
+-- -------------------------------------------------------------------------- --
+-- ReintroTxsConfig
+
+data ReintroTxsConfig = ReintroTxsConfig
+    deriving (Show, Eq, Generic)
+
+makeLenses ''ReintroTxsConfig
+
+defaultReintroTxsConfig :: ReintroTxsConfig
+defaultReintroTxsConfig = ReintroTxsConfig
+
+instance ToJSON ReintroTxsConfig where
+    toJSON _ = object []
+
+instance FromJSON (ReintroTxsConfig -> ReintroTxsConfig) where
+    parseJSON = withObject "ReintroTxsConfig" $ const (return id)
+
+pReintroTxsConfig :: MParser ReintroTxsConfig
+pReintroTxsConfig = pure id
+
 -- -------------------------------------------------------------------------- --
 -- Chainweb Configuration
 
 data ChainwebConfiguration = ChainwebConfiguration
     { _configChainwebVersion :: !ChainwebVersion
     , _configNodeId :: !NodeId
     , _configMiner :: !(EnableConfig MinerConfig)
+    , _configReintroTxs :: !(EnableConfig ReintroTxsConfig)
```

Comment:
I wonder if it would make sense to introduce a `MempoolConfig`data type to bundle all all Mempool related settings.

```Haskell
data MempoolConfig = MempoolConfig
    { _mempoolConfigP2p :: !MempoolP2pConfig
    , _mempoolConfigReintroTx :: !(EnableConfig ReintroTxsConfig) -- or, alternatively, just !Bool
    }
```

Maybe it's too early at this point, but if we keep adding mempool options we should do it.

Reply:
There already is an InMemConfig type (and at this point we are only using the InMem variant -- the other was removed) :

data InMemConfig t = InMemConfig {
    _inmemTxCfg :: {-# UNPACK #-} !(TransactionConfig t)
  , _inmemTxBlockSizeLimit :: {-# UNPACK #-} !Int64
  , _inmemReaperIntervalMicros :: {-# UNPACK #-} !Int
  , _inmemEnableReIntro :: !Bool
}

I didn't realize i didn't need the additional type for interfacing with the CLI.